LineSorter/source.extension.vsixmanifest

```diff
@@ -13,13 +13,16 @@
         <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Pro" />
         <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.IntegratedShell" />
         <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,17.0)" />
+        <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Community">
+            <ProductArchitecture>amd64</ProductArchitecture>
+        </InstallationTarget>
```

Comment:
Oh, by the way. I think the same thing should be done for the `Enterprise` version

Reply:
`ProductArchitecture` defaults to `x86`, so it's not necessary to add it for pre-2022 versions. But it doesn't hurt, so if you want it for clarity, that's no problem.

Reply:
As far as I know, you only need to add the lowest version type that the extension will work on. So adding Community, implies that it will work on Pro and Enterprise.

Reply:
Wish I knew that back in the days xD Ok, got it